File: BubbleSort.java
//Author: Benjamin Gantenbein

import java.util.Arrays;

public class BubbleSort{


  public static int[] bubblesort(int[] array){

      for(int i= 0; i < array.length; i++){
        for(int j = i; j < array.length; j++){
            if(array[i] > array[j]){
              int temp = array[i];
              array[i] = array[j];
              array[j] = temp;
            }

        }
      }
      return array;


  }



  public static void main(String[] args){


          int[] array = {900, 1, 3, 4, 6, 72, -7, 9};


          System.out.println(Arrays.toString(bubblesort(array)));


  }

}

File: Sorting/BubbleSort.py
def BubbleSort(array):
    length = len(array)

    for i in range(length):
        for j in range(i, length):
            if array[i] > array[j]:
                array[j], array[i] = array[i], array[j]

array = [-72, 232134, -1, 33, 55555, 234, 20000]
for i in range(len(array)):
    print(array[i], end=" ")

BubbleSort(array)
print(" ")
for i in range(len(array)):
    print(array[i], end=" ")
